server/services/routeUrls.ts

```diff
@@ -5,9 +5,9 @@ export default {
     view: '/prisoner/:prisonerNumber/non-associations',
     prisonerSearch: '/prisoner/:prisonerNumber/non-associations/add/search-prisoner',
     add: '/prisoner/:prisonerNumber/non-associations/add/with-prisoner/:otherPrisonerNumber',
-    edit: '/prisoner/:prisonerNumber/non-associations/edit/with-prisoner/:otherPrisonerNumber',
-    close: '/prisoner/:prisonerNumber/non-associations/close//with-prisoner/:otherPrisonerNumber',
-  },
+    close: '/prisoner/:prisonerNumber/non-associations/close/:nonAssociationId',
+    update: '/prisoner/:prisonerNumber/non-associations/update/:nonAssociationId',
```

Comment:
it's probably easier to use the non-association ID in the url rather than hope that no-associations will always be unique by a pair of prisoner numbers

Reply:
Do you need the prisoner number in the URL? Wouldn't be nice and clean to just have the non-association ID? e.g. something like `/non-associations/:id/close`, etc

Reply:
that's a fair question. i was trying to mimic the information architecture model so that the urls stem from the prisoner profile like the breadcrumbs etc do. my plan is to have it either redirect or 404 if the prisoner number does not match either side of the nonna.

but at this point, the pages do nothing! and we probs should talk as a team about what they ought to be anyway

Reply:
yeah no worries, I don't have a strong opinion on this anyway :)